graphix/command.py

```diff
@@ -0,0 +1,112 @@
+"""Data validator command classes."""
+
+from pydantic import BaseModel
+from typing import Union, Literal, List, Tuple
+from graphix.pauli import Plane
+import enum
+
+Node = int
+
+class CommandKind(str, enum.Enum):
```

Comment:
(comment by @EarlMilktea)  - it does not make sense for this to inherit from `str` - e.g.  this should not inherit [string methods](https://docs.python.org/3/library/stdtypes.html#string-methods).